compute_endpoint/packaging/debian/changelog.in.in

```diff
@@ -1,8 +1,8 @@
-@PACKAGE_NAME@ (@PACKAGE_VERSION@.@distro@) @distro@; urgency=low
+@PACKAGE_NAME@ (@PACKAGE_VERSION@-1.@distro@) @distro@; urgency=low
```

Comment:
I'm not famiilar with the @-1.@ syntax, is this referring to the 1st?  I don't see it referenced elsewhere.

Reply:
This is a Debian-ism.  I actually like it, but only after I understood its purpose.

For posterity, one of the major philosophies driving Debian packaging is keeping a separation between the _developer's_ version and the _packager's_ version.  That `-1` is the packaging version.  We might change that at some point if we have to rebuild the same version [for some reason] of a package after having released it.  But that will typically always be `-1`.